### PR TITLE
Add French (Canadian) translation

### PR DESCRIPTION
**Pull request by UsernameSwift**

### CODE REVIEW

File: public/locales/fr/common.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "other": "Autres Ressources"
+  },
+
+  "locations": {
+    "home": "Accueil",
+    "docs": "Documentation",
+    "forums": "Forum",
+    "twitter": "Twitter",
+    "mastodon": "Mastodon",
+    "github": "Github",
```
Comment (TheEvilSkeleton) [inline public/locales/fr/common.json:20]:
```suggestion
    "github": "GitHub",
```

File: public/locales/fr/common.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "discord": "Discord",
+    "matrix": "Matrix",
+    "open_collective": "Open Collective",
+    "contributing_guidelines": "Directrices Pour les Contributions",
+    "contributor_covenant": "Code de Conduite 'Contributor Covenant'",
```
Comment (TheEvilSkeleton) [inline public/locales/fr/common.json:25]:
```suggestion
    "contributor_covenant": "Convention des Contributeurs'",
```

File: public/locales/fr/home.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "timeshift": {
+    "tag": "Sûr",
+    "title": "Sauvegardes Automatiques",
+    "description": "Sauvegardez et Restaurez votre système. Démarrez dans des aperçus Btrfs facilement."
+  },
+  "amethyst": {
+    "tag": "Quartz rouillé?",
+    "title": "Amethyst",
+    "description": "Amethyst est un assistant de AUR et emballage pour Pacman qui est rapide, efficient et légère. C'était fabriqué pour Crystal, mais c'est compatible avec n'importe quelle distribution basée sur Arch."
```
Comment (TheEvilSkeleton) [inline public/locales/fr/home.json:15]:
```suggestion
    "description": "Amethyst est un assistant de AUR et emballage pour Pacman qui est rapide, efficient et légère. C'était fabriqué pour Crystal, mais c'est compatible avec n'importe quelle distribution basée sur Arch Linux."
```

File: public/locales/fr/meta.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "documentation": "Documentation est maintenu par la communauté! Les traductions peuvent contenir information qui n'est pas mise à jour, ou qui est incorrecte."
```
Comment (TheEvilSkeleton) [inline public/locales/fr/meta.json:2]:
```suggestion
  "documentation": "La documentation est maintenue par la communauté! Les traductions peuvent contenir information qui n'est pas mise à jour, ou qui est incorrecte."
```

File: public/locales/fr/navbar.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "open_main_menu": "Ouvrez le Menu Primaire",
+  "switch_theme": "Changez le Thème",
+  "change_language": "Changez la Langue"
```
Comment (TheEvilSkeleton) [inline public/locales/fr/navbar.json:4]:
```suggestion
  "open_main_menu": "Ouvrir le Menu Primaire",
  "switch_theme": "Changer le Thème",
  "change_language": "Changer la Langue"
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Comment by mrshmllow** [comment]
please update next.config.mjs and add "fr-CA"

File: public/locales/fr/home.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "timeshift": {
+    "tag": "Sûr",
+    "title": "Sauvegardes Automatiques",
+    "description": "Sauvegardez et Restaurez votre système. Démarrez dans des aperçus Btrfs facilement."
+  },
+  "amethyst": {
+    "tag": "Quartz rouillé?",
+    "title": "Amethyst",
+    "description": "Amethyst est un assistant de AUR et emballage pour Pacman qui est rapide, efficient et légère. C'était fabriqué pour Crystal, mais c'est compatible avec n'importe quelle distribution basée sur Arch Linux."
```
Comment (Antiz96) [inline public/locales/fr/home.json:15]:
```suggestion
    "description": "Amethyst est un assistant AUR et une surcouche pour Pacman rapides, efficaces et légers. Il a été développé pour Crystal, mais est compatible avec n'importe quelle distribution basée sur Arch Linux."
```

File: public/locales/fr/home.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    "title": "Onyx inclus",
+    "description": "Installer Crystal avec sa propre experience de bureau, une session de GNOME avec une disposition familière. Ou, choisissez votre environnment préféré!"
+  },
+  "timeshift": {
+    "tag": "Sûr",
+    "title": "Sauvegardes Automatiques",
+    "description": "Sauvegardez et Restaurez votre système. Démarrez dans des aperçus Btrfs facilement."
+  },
+  "amethyst": {
+    "tag": "Quartz rouillé?",
```
Comment (Antiz96) [inline public/locales/fr/home.json:13]:
```suggestion
    "tag": "Quartz rouillé ?",
```

File: public/locales/fr/meta.json [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{
+  "documentation": "La documentation est maintenue par la communauté! Les traductions peuvent contenir information qui n'est pas mise à jour, ou qui est incorrecte."
```
Comment (Antiz96) [inline public/locales/fr/meta.json:2]:
```suggestion
  "documentation": "La documentation est maintenue par la communauté ! Les traductions peuvent contenir des informations obsolètes ou incorrectes."
```

File: public/locales/fr/navbar.json [hunk 9]
```diff
@@ -0,0 +1,5 @@
+{
+  "open_main_menu": "Ouvrir le Menu Primaire",
```
Comment (Antiz96) [inline public/locales/fr/navbar.json:2]:
```suggestion
  "open_main_menu": "Ouvrir le menu principal",
```

File: public/locales/fr/navbar.json [hunk 10]
```diff
@@ -0,0 +1,5 @@
...
+  "switch_theme": "Changer le Thème",
```
Comment (Antiz96) [inline public/locales/fr/navbar.json:3]:
```suggestion
  "switch_theme": "Changer le thème",
```

File: public/locales/fr/navbar.json [hunk 11]
```diff
@@ -0,0 +1,5 @@
...
+  "change_language": "Changer la Langue"
```
Comment (Antiz96) [inline public/locales/fr/navbar.json:4]:
```suggestion
  "change_language": "Changer la langue"
```

File: public/locales/fr/common.json [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+    "docs": "Documentation",
+    "forums": "Forum",
+    "twitter": "Twitter",
+    "mastodon": "Mastodon",
+    "github": "GitHub",
+    "discord": "Discord",
+    "matrix": "Matrix",
+    "open_collective": "Open Collective",
+    "contributing_guidelines": "Directrices Pour les Contributions",
+    "contributor_covenant": "Convention des Contributeurs'",
```
Comment (Antiz96) [inline public/locales/fr/common.json:25]:
```suggestion
    "contributor_covenant": "Convention des contributeurs'",
```

File: public/locales/fr/home.json [hunk 13]
```diff
@@ -0,0 +1,17 @@
+{
+  "onyx": {
+    "tag": "Beau",
+    "title": "Onyx inclus",
+    "description": "Installer Crystal avec sa propre experience de bureau, une session de GNOME avec une disposition familière. Ou, choisissez votre environnment préféré!"
+  },
+  "timeshift": {
+    "tag": "Sûr",
```
Comment (Antiz96) [inline public/locales/fr/home.json:8]:
```suggestion
    "tag": "Fiable",
```

File: public/locales/fr/home.json [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+    "title": "Sauvegardes Automatiques",
+    "description": "Sauvegardez et Restaurez votre système. Démarrez dans des aperçus Btrfs facilement."
```
Comment (Antiz96) [inline public/locales/fr/home.json:10]:
```suggestion
    "description": "Sauvegardez et Restaurez votre système. Démarrez facilement sur des instantanés Btrfs."
```

**Review by Antiz96 — CHANGES_REQUESTED** [review]
Here are some improvements suggestions.

**Comment by UsernameSwift** [comment]
I've resolved the merge conflict that was present, and having had changes made by people who know the language far better than I do, I'd say that the additions here are complete.

**Pull request merged**